source/guide_php-fusion.rst

```diff
@@ -0,0 +1,88 @@
+.. author:: DwB <dirk@babig.de>
+.. highlight:: console
+
+.. tag:: lang-php
+.. tag:: web
+.. tag:: cms
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/php-fusion.png
+      :align: center
+
+##########
+PHP-Fusion
+##########
+
+PHP-Fusion_ is a CMS written in PHP and distributed under the GN AGPL v3 licence.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * PHP_
+  * :manual:`MySQL <database-mysql>`
+  * :manual:`domains <web-domains>`
+
+Prerequisites
+=============
+
+We're using PHP_ in the stable version 7.1:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '7.1'
+ [isabell@stardust ~]$
+
+MySQL
+-----
+ 
+.. include:: includes/my-print-defaults.rst
+
+Domain
+------
+
+Your URL needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+First get the PHP-Fusion source code from Github_:
+
+::
+
+  [isabell@stardust ~]$ git clone https://github.com/PHP-Fusion/PHP-Fusion html
+  Cloning into 'html'...
+  remote: Enumerating objects: 140, done.
+  remote: Counting objects: 100% (140/140), done.
+  remote: Compressing objects: 100% (121/121), done.
+  remote: Total 88715 (delta 54), reused 62 (delta 18), pack-reused 88575
+  Receiving objects: 100% (88715/88715), 111.94 MiB | 14.20 MiB/s, done.
+  Resolving deltas: 100% (61743/61743), done.
+  [isabell@stardust ~]$
+
+  
+Finishing installation
+======================
+
+Point your browser to your domain, for example https://isabell.uber.space,
+and follow the installation instructions.

```

Comment:
I only get an error message

`Fatal error: Uncaught Error: Class 'Defender' not found in /var/www/virtual/testnico/html/includes/dynamics/includes/form_select.php:344 Stack trace: #0 /var/www/virtual/testnico/html/includes/classes/PHPFusion/Installer/Steps/Introduction.inc(84): form_select('localeset', '', 'English', Array) #1 /var/www/virtual/testnico/html/includes/classes/PHPFusion/Installer/Steps/Introduction.inc(29): PHPFusion\Steps\InstallerIntroduction->__Index() #2 /var/www/virtual/testnico/html/includes/classes/PHPFusion/Installer/Install.core.inc(351): PHPFusion\Steps\InstallerIntroduction->__view() #3 /var/www/virtual/testnico/html/includes/classes/PHPFusion/Installer/Install.core.inc(330): PHPFusion\Installer\Install_Core->get_InstallerContent() #4 /var/www/virtual/testnico/html/install.php(22): PHPFusion\Installer\Install_Core->install_phpfusion() #5 {main} thrown in /var/www/virtual/testnico/html/includes/dynamics/includes/form_select.php on line 344`

Reply:
Yes, that's a bit of shame, that the developers of PHP-Fusion just don't fix this issue, although it was opened and closed with the argument "it's development" -> https://github.com/php-fusion/PHP-Fusion/issues/2212

I think I'll rewrite the whole installation routine using the latest stable version.